src/design-system/layout/footer/footer.module.scss

```diff
@@ -1,3 +1,54 @@
 .footer {
   display: flex;
+  flex-direction: column;
+  height: 200px;
+  font-family: "Poppins", sans-serif;
+}
+
+.navLinks {
+  display: flex;
+  list-style-type: none;
+  width: 90%;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 60px 0 20px;
+  border-bottom: 2px solid white;
+
+  li {
+    margin: 0px 20px;
+    font-size: 20px;
+
+    &:first-of-type {
+      margin-left: 0px;
+    }
+  }
+}
+
+.bottomLinks {
+  display: flex;
+  list-style-type: none;
+  width: 90%;
```

Comment:
niepotrzebnie zdublowane style, jak złapiesz <Container> (komentarz wyżej) to potem nie musisz definiować tutaj tego width, max-width itd